File: GeeksForGeeks/School/Reverse_a_string.java
import java.util.Scanner;

class GFG {
    public static void main(String[] args) {
        
        Scanner scan = new Scanner(System.in);

        int testCase = scan.nextInt();
        scan.nextLine();
        while (testCase != 0) {
            String str = scan.nextLine();

            char[] strArr = str.toCharArray();
            int i = 0, j = strArr.length - 1;
            char c;
            while (i < j) {
                c = strArr[j];
                strArr[j] = strArr[i];
                strArr[i] = c;
                ++i;
                --j;
            }

            System.out.println(String.valueOf(strArr));
            --testCase;
        }
        scan.close();
    }
}

File: GeeksForGeeks/School/Reverse_a_string.cpp
#include <iostream>
using namespace std;

int main() {
    int test_case;
    cin >> test_case;

    while (test_case--) {
        string str;
        cin >> str;
        
        int i = 0, j = str.length() - 1;
        while (i < j) {
            char c = str[j];
            str[j] = str[i];
            str[i] = c;
            ++i;
            --j;
        }
        cout << str << endl;
    }
    return 0;
}